docker.d/init_worker.sh

```diff
@@ -10,15 +10,22 @@ CERT_DIR=/app/files
 function import_cert() {
         JARSIGNER_KEY_STORE_NAME=$1
         JARSIGNER_KEY_STORE_CERTIFICATE=$2
+        echo "Importing $JARSIGNER_KEY_STORE_CERTIFICATE as $JARSIGNER_KEY_STORE_NAME into $JARSIGNER_KEY_STORE"
         keytool \
           -importcert \
           -noprompt \
           -alias $JARSIGNER_KEY_STORE_NAME \
           -file $JARSIGNER_KEY_STORE_CERTIFICATE \
           -keystore $JARSIGNER_KEY_STORE \
           -trustcacerts \
+          -storetype jks \
           -srcstorepass $JARSIGNER_KEY_STORE_PASSWORD \
           -deststorepass $JARSIGNER_KEY_STORE_PASSWORD
+        keytool \
```

Comment:
Will listing the keys in the store leak any valuable information in logs?

Reply:
The keys are public and the boot logs are not, we should be safe.